scripts/specializations/CpAIFieldWorker.lua

```diff
@@ -285,7 +285,12 @@ function CpAIFieldWorker:replaceAIFieldWorkerDriveStrategies(jobParameters)
         spec.driveStrategies = {}
     end
     local cpDriveStrategy
-    if AIUtil.getImplementOrVehicleWithSpecialization(self, Combine) and not AIUtil.hasChildVehicleWithSpecialization(self, VineCutter) then
+    local course = self:getFieldWorkCourse()
+    if course and course:hasVineNodesOnTheField() then 
```

Comment:
Not sure if this is the best way to decide if this strategy is needed, couldn't we use the same logic which decides which course generator parameters to show?

Reply:
Not all vine implements are unique, for example the sprayer, which is has the same as a normal sprayer. Basically we need a solution for only using this strategy, when a vine field is present. Another possiblity would be saving a flag, like "isVineFieldWorkCourse" in the course class.